cleanup-pr-branch

```diff
@@ -49,7 +55,7 @@ main(){
 		echo "Branch delete success!"
 		exit 0
 	fi
-	exit 78
+	exit NO_BRANCH_DELETED_EXIT_CODE
```

Comment:
does this need a $

Reply:
You're absolutely right! I failed to copy paste correctly when I moved it from my test file into the real script.